knpu/ep3-webpack/caching-versioning.md

```diff
@@ -57,6 +57,23 @@ add `useVersioning`. If it's on, use `[name].[contenthash:6].css`:
 
 [[[ code('2b589790b0') ]]]
 
+***TIP
+Hey! If you're using Webpack 4, you'll need to make the following change
+
+```javascript
+   const useVersioning = true;
+   // ...
+   const webpackConfig = {
+       // ...
+       plugins: [
+           new ExtractTextPlugin(
+               useVersioning ? '[name].[md5:contenthash:hex:6].css' : '[name].css'
```

Comment:
What's wrong with using `[name].[contenthash:6].css` in Webpack4? The new filename *does* still use those same placeholders. We also use basically the old string in Encore with Webpack 4: https://github.com/symfony/webpack-encore/blob/master/lib/plugins/mini-css-extract.js#L22